tests/foreman/cli/test_errata.py

```diff
@@ -198,11 +360,19 @@ def test_negative_install_by_hc_id_without_errata_info(self):
 
         @Assert: Error message thrown.
 
-        @caseautomation: notautomated
-
         """
+        with self.assertRaises(CLIReturnCodeError) as context:
+            HostCollection.erratum_install({
+                'id': self.host_collection['id'],
+                'organization-id': self.org['id'],
+            })
+        self.assertIn(
+            "Error: option '--errata' is required",
```

Comment:
Yeah, I talked about that a lot in the past, but it seems that everyone agree that it is fine to use it. There are so many occurrences of that approach through the code, that we should not block it till it is good enough way to check functionality or there is no visible alternative